analysis/parser/RawTextParser/README.md

```diff
@@ -23,6 +24,8 @@ generates cc.json from projects or source code files
                              include build, target, dist, resources and out folders
                                as well as files/folders starting with '.'
   -e, --exclude=<exclude>    exclude file/folder according to regex pattern
+  -f, --file-extensions[=<file-extensions>...]
```

Comment:
-f could be misleading. I would expect it can be used for specifying an input file

Reply:
Would it be better to rename it to file-type in the help and use `-t` or `--file-types`?

Reply:
In that case I would rather go with -e, --extensions.

Reply:
`-e`, `--exclude` is already in use to exclude files by a regex pattern.
Two other parsers (merge and sourcecodeparser) also use a `-f` flag (for `--move-file` and `--format`) so I think we should either allow it here too or change it for them as well.

 Alternatively we could use `-fe` for `--file-extensions`, similar to how some parser use `-nc` as short for `--not-compressed`.

Reply:
It is hard to name these things.
What do you think? `-fe` is the best way I think.